main/src/ca/uwaterloo/flix/language/phase/Safety.scala

```diff
@@ -255,9 +255,6 @@ object Safety {
     case Expression.Par(exp: Expression.Tuple, _) =>
```

Comment:
I would like to have just one Par case where you match on the inner expression and then a comment that explains what is happening :)